### ISSUE TITLE
Reverting unit balance to a previous version

### ISSUE THREAD

**Issue by Pentarctagon** [wesnoth/wesnoth#9392]
So as a follow up to #9389 which got a pretty clear response: what would reverting back to previous unit stats actually mean? I've seen different people mention using the 1.12, 1.14, or 1.16 unit stats. Some people have mentioned using version 1.1X but keeping the uncontroversial changes from 1.18 (though at this point it's a bit hard to imagine that such a change actually exists). Also wouldn't reverting most/all of the 1.18 changes cause a huge amount of additional rework at this point?

**Comment by Dalas121** [comment]
I think it would be better to approach this question on a unit-by-unit basis, rather than make more bulk changes. As you pointed out, this would cause significant additional headache - and IMO many of 1.18's changes weren't issues.

For example, the 1.18 elf buffs were widely unpopular, but have already been (and/or are being) changed. And while the arcane rework wasn't universally loved, I generally think it was a good thing for SP gameplay.

If there are specific changes that we or community members would like to see reversions/adjustments to, perhaps it's best for players to create separate issues, so general community sentiment can be gauged before any specific PR is made.

**Comment by ForestDragon-wesnoth** [comment]
I would personally support reverting to 1.16 stats as a baseline

**Comment by Roge-Tebnelok** [comment]
At least reverting to old arcane resistances.

**Comment by kameramen0106** [comment]
I agree that we should revert back to 1.16 for the basics. I am personally convinced that the unit balance in Wesnoth was complete at that point. This is my opinion based on my subjective view and community tendencies. Well, of course, I am not going to claim that all the changes in 1.18 are bad.

Perhaps the main purpose of the update is to keep Wesnoth up to modern standards and to attract new players.

However, the changes for this purpose have caused opposition from existing players and creators. This should not be ignored. Also, I would like to add that some new players, like myself, are attracted to the less modern aspects of the game.

**Comment by amakriLexa04** [comment]
> I would personally support reverting to 1.16 stats as a baseline, it was the latest version that was balanced around the old philosophy of not making too radical changes in updates.

I support reverting the balance to 1.16 and closing this topic forever, because the recent changes that seem to be made just for the sake of change are often very disappointing.

However, it should be taken into account whether the balance of SP campaigns has changed in accordance with the changes in unit stats in new versions. Personally I don't think anyone has audited the balance of old campaigns for quite some time, so it shouldn't be a problem to revert the balance changes. However, if such an audit has been conducted, then it is necessary to consider in more detail how the return to 1.16 will affect the existing balance in the campaigns.

**Comment by xSkyEndx** [comment]
1.14 is bit better. I explained the changes.
https://forums.wesnoth.org/viewtopic.php?t=58621&start=150

**Comment by MechanicalDragon963** [comment]
I agree with balance changes in 1.18 got too far and exaust many energy of developers and community for discussions. I think it's fine to revert into 1.16 unit stats statement ad close the topic at least for a while.

**Comment by knyghtmare** [comment]
> I think it may be better to approach this question on a unit-by-unit basis, rather than make more bulk changes. As you pointed out, bulk changes will cause significant additional headache - and IMO many of 1.18's changes weren't issues.

Yes, I agree on this. 

Not every balance change in 1.18 branch was received with a negative sentiment.

I believe we should start with the release of 1.18.x as a base and then list up all the units that had a change in stats, and then discuss/review them all...one by one.
If they are okay as is, we pass them (that will be the case for 90% of the current core units)

For the remaining 10%, I believe those will be the ones in need of the collective discussion and finalisation of new stats. From my understanding, the units that might fall in this 10% are the mage-class units (I saw a stats change PR about them recently by Dalas121), and maybe the HI, Orc Archer, Huntsman, Ranger and Dune Burner.

>  I think it's fine to revert into 1.16 unit stats statement ad close the topic at least for a while.

I would treat this as a last resort option. As in, if done, I suppose it might resolve the majority complaints. However, the case by case discussions should be taken first.

**Comment by sevu** [comment]
I would categories the changes done for 1.18 as:
* arcane resistance change
* mage xp change
* other unit xp changes
* higher lvl units gold changes
* changes to mp recruitable level 1's
* elf changes

Of these I value very much the gold changes, while I would only for the elf changes prefer 1.16 stats.

**Comment by AnekronCZ** [comment]
From my SP focused perspective I think 1.18 is mostly fine as long as the few well known issues are addressed (such as the over-buffed elves or the overly cheap Slurbow; something that Hejne wasn't opposed to addressing, btw). Arcane resistance seems to be controversial among some as well, though I don't feel it negatively affected the game personally.

I'm not opposed to 1.16 balance in principle, though I don't believe the best course of action is to simply roll back everything.

I'm not at all in favour of reverting to 1.14 balance, at least definitely not for Dunefolk, since the faction had a massive revamp in 1.16.

**Comment by Discontinuum** [comment]
I like 1.16 unit stats. For me the two problems of 1.18 were a series of too strong breaking changes and a lot of small not very well explained xp-hp tweaks changes. Because of it I decided to base Afterlife Era on 1.16 in order to continue balancing without worrying too much about the mainline changes effects. 

Balance of 1.14 may appeal more to Ladder era players because of different approaches to Elvish Shaman and Merman Hunter, for me 1.16 is more comfortable. I also like Dunefolk 1.16 more than Dunefolk 1.14, while ladder is generally indifferent to Dunefolk.

I agree that it makes sense to discuss changes of 1.18 in groups or individually and possibly leave some of them in place.

A brief summary of my opinions on the most important changes for me:

* A lot of changes to XP (and some HP) of level 1 and level 2 units. Those may seem pretty insignificant but they are pain for Ladder because many normal strategies are disrupted. I've seen complains about Loyalist Mage, also SkyEnd made a pretty good overview on the forums. So I'd prefer mostly going back to 1.16 here.
* Saurian Soothsayer buffed MP and the subsequent removal from the Drakes leader pool, very strange buff.
* OP Elves, enough been said about them. I would also note increased MP of Elvish Outrider, feels like too much, he became faster than Gryphon Riders. That one also needs attention in regard of "Outride the Outriders" scenario of SoF.
* Arcane rework. I don't really like it, it diminished arcane into "just another magical type of damage" and made a special note about it pretty much outdated. I would revert it or at least make it closer to 1.16: return 20% arcane for humans (and increased arcane res of the White Mage line and Paladin) and make it at least -30% for skeletons.
* Orcish Crossbowman. It received a significant buff: very fast leveling to lvl 3 and increased melee. If not going back to 1.16 completely, I would nerf him back a bit, more xp to level, a bit less melee damage.

Probably forgot something, will add later if remember.

Obviously we should  keep new Saurian Augurs 3 levels, and I believe keep the advancement of Orcish Slayer to Orcish Nightblade.

**Comment by Dalas121** [comment]
> Of these I value very much the gold changes

I agree. I'd prefer to keep these if possible.


> A lot of changes to XP (and some HP) of level 1 and level 2 units.

The level 1 XP/HP changes were fairly minor from a SP perspective, so I have no issue reverting them to help MP. IIRC the level 2 changes were mostly XP reductions, which I also support reverting to delay maxing-out on armies in SP.


> I would also note increased MP of Elvish Outrider, feels like too much

I agree. I'd also be in favor of moving some power from the Rider/Outrider's (not Scout's) melee to their ranged damage - both to give them a more defined SP combat role besides scouting, and also because horse archery seems more on-theme for elves.


> Arcane rework. I don't really like it, it diminished arcane into "just another magical type of damage"

I think there's two competing issues with arcane damage. First, thematically, arcane should feel special and unique rather than just another kind of magic. Second, arcane shouldn't be so fantastically good against undead that all you recall are arcane/impact units - we want to encourage unit variety. Unfortunately these are rather at odds with each-other.

I have a couple not-thought-through suggestions, to hopefully reconcile these competing goals for arcane damage. What are your opinions on each of these?

- 20% humans/ghouls/animals/etc and -30% skeletons/liches, but only -10% for corpses (identical to ghosts). Keeping arcane as the best option vs undead, but letting campaign designers use ghouls+corpses to "counter" arcane-heavy recall lists.

- 0% humans/ghouls/animals/etc and -50% skeletons/liches/corpses, but reducing damage for arcane attacks. Keeping arcane clearly unusual, but not making arcane units so strong vs undead that they're all you recall.

- -10% or -0% impact/fire resist for skeletons/liches. I'm guessing this would have serious MP consequences though.


> Cavalryman - blade resistance changed from 30% to 20%, 

Right now, the Cavalryman has different blade resist than the Dragoon/Cavlier, who still have 30%. For consistency's sake, I'd prefer to either revert the Cavalryman or nerf the Dragoon/Cavalier.


As it sounds like we'll be making major changes to many units' stats, I wonder if this could be a good opportunity to reconcile some of Ladder Era's more minor changes with mainline/default. I think changes like "Guardsman: +1 Ranged Damage" or "Heavy Infantryman: Retains its original -10% Cold Resist from Wesnoth 1.14" are small enough that we could easily bring them to mainline, which would help to both 1) make the MP playerbase less fragmented and 2) (presumably) make mainline more balanced.

**Comment by ForestDragon-wesnoth** [comment]
> I agree. The Rider and especially Outrider also got significant damage buffs in 1.18. Rather than reverting those completely, I'd be in favor of moving some power from their melee to their ranged damage - both to give them a more defined SP combat role besides scouting, and also because horse archery seems more on-theme for elves.

Compared to 1.16, outrider got a very significant ranged bonus (from 8-3 to 11-3), though I do think something like 7-4 melee 9-3 ranged 10mp could be one of the potential compromises.

> What's your opinion about 20% humans/ghouls/animals/etc and -30% skeletons/liches, but only -10% for corpses (identical to ghosts)? That would keep arcane as a distinct and best option vs undead, but still allow campaign designers to use ghouls+corpses to discourage arcane/impact spam.

walking corpse/soulless being more resistant to arcane won't really affect the situation that much, for a normal lvl2-3 army they are basically insects. It could however have ramifications for MP balance.

**Comment by hladkyjakub** [comment]
I'm against.
Those changes were once made, we adapted already, why in the deepest pit of bloody hell would anyone want to take them back?
If I had to choose beween reverting and branching units, I would go for branching.

I don't want Wesnoth to be "Historic" game, or same as I played it for the first time.
I want the game to be fun.

**Comment by DuncanusDillius** [comment]
Pesonally, I think we should revert to 1.14 or 1.16 stats. But we shouldn't leave it at that, instead we should improve them on a unit-by-unit basis, and see what we can improve, and what should stay the same. Though, this should be done via community voting, maybe a poll on the forums would be nice.

**Comment by ForestDragon-wesnoth** [comment]
> Pesonally, I think we should revert to 1.14 or 1.16 stats. But we shouldn't leave it at that, instead we should improve them on a unit-by-unit basis, and see what we can improve, and what should stay the same.

Yeah, some individual changes might be fine to keep, as long as 1.16 is treated as the baseline and the question is more "what do we need to keep from 1.18" rather than "what do we remove from 1.18, and keep everything else"

> Though, this should be done via community voting, maybe a poll on the forums would be nice.

I think a forum discussion is the best format for that (like the 1.20 balance thread)

**Comment by Dalas121** [comment]
> walking corpse/soulless being more resistant to arcane won't really affect the situation that much, for a normal lvl2-3 army they are basically insects.

I disagree. I make heavy use of WC/Soulless in a number of EI scenarios, including the final few, and I feel they provide a meaningful threat.  A Soulless does the same damage as a Skeleton, but is cheaper, fearless, and vulnerable to neither impact nor fire.

WC / Soulless can also be "customized" with variations, if the theme fits your campaign. Iron Maulers are surprisingly weak against a horde of mounted, horse, wose, or scuttler WC (impact damage plus impact resist), and drakish WCs can be a terror for fire mages.

Even their low hp can be a benefit, as a powerful melee defender may often slay their attacker, opening the hex back up for further attackers.

> It could however have ramifications for MP balance.

As the Ghost is MP's only level-1 arcane unit, I don't expect this to have significant MP balance ramifications - especially not in comparison to the much more impactful proposal of 20% human / -30% skeleton.

**Comment by ForestDragon-wesnoth** [comment]
> As the Ghost is MP's only level-1 arcane unit, I don't expect this to have significant MP balance ramifications - especially not in comparison to the much more impactful proposal of 20% human / -30% skeletal.

ok, good point on the gameplay side

**Comment by Pentarctagon** [comment]
@hladkyjakub comment was what I was referring to by:
> Also wouldn't reverting most/all of the 1.18 changes cause a huge amount of additional rework at this point?

I do think this point needs to be considered.

It also feels pretty extreme to me to say that everything should be changed back to be exactly the same as 1.16 or even 1.14, so I think the approach in @Dalas121 and @knyghtmare's first comments make more sense.

---

Doesn't the Ladder use its own era? So I'm not sure why the Ladder is really relevant here since as far as I know they've already been using their own units since long before 1.18.

---

> Though, this should be done via community voting, maybe a poll on the forums would be nice.

No, this isn't going to be done by open voting. If there are specific, skilled people who want to be part of a small balancing team that could be done (and was already brought up once before in fact) but I can't imagine that determining unit stats via forum polls will result in good balance.

**Comment by Mirion147** [comment]
As far as reverting, I'm (for however much it's worth) very much against it. I'm not against changing the game (especially the stats), I'm not against adding things to the game, I'm not against updating older content. What I am against is removing content completely (AOI), adjustments that don't make sense thematically (Elven Archer gaining blade resist), and adjustments that are just ridiculous (the Elven Shaman line and Orcish crossbowman buffs of 1.18)

The way that I view it, having stats that change (minorly) is not a bad thing. I don't mind seeing how it impacts my campaign, it may make players feel like they have a better chance, or see something as a bigger challenge, and it's made the campaign simple then 2 things are true, 1 I need to adjust the campaign, and 2 that's okay. There may still be changes like the Elven Shaman line which are just way too extreme, and I'm not in favor of massive buffs/nerfs to any units, but just having a meta that changes is something I find interesting. 

By reverting back to old stats you're solidifying that the game will never change and once I've played a campaign there's not much reason to play it again other than for nostalgia or made up challenges. It feels boring and flat to me and would be a step in the wrong direction for me.

**Comment by Dalas121** [comment]
> Doesn't the Ladder use its own era? So I'm not sure why the Ladder is really relevant here since as far as I know they've already been using their own units since long before 1.18.

I know very little about MP Wesnoth and I don't think this is the most important issue at hand, but I do feel that Ladder Era - and other modified default eras - represent a split in the community. Wesnoth has a limited amount of MP players, so in theory any division in the community make games harder to find for both sub-communities.

In addition, I assume that Ladder Era is very well-balanced. In theory, porting some of those changes over would also help make Default Era more balanced (though I'm sure the reality is a little more complex).

**Comment by CelticMinstrel** [comment]
For what it's worth, I think it's crazy that people are talking about reverting to 1.16 stats. You expect me to believe that _every single change_ made between 1.16 and 1.18 is bad? For the most part, I don't think I care about most of the changes, so I'd probably even be fine with not reverting anything… but that said, I don't really have a problem with some of the specific reverts that Dalas and others were discussing, either.

> Though, this should be done via community voting, maybe a poll on the forums would be nice.

No. Voting is good for some kinds of decision-making. It is _not_ good for designing a game.

> I can't imagine that determining unit stats via forum polls will result in good balance.

Yeah, there's no way. Even _if_ balancing were to be handled by a team, voting amongst themselves would not be the correct method.

**Comment by Mirion147** [comment]
While I don't think that polls should determine anything, I do think it's a good thing to have polls to gauge where people stand on certain topics

**Comment by CelticMinstrel** [comment]
No, not really. Not for anything that truly _matters_. If it's something like "which of these optional new fauna units does the community want most", then sure.

**Comment by Roge-Tebnelok** [comment]
> I like 1.16 unit stats. For me the two problems of 1.18 were a series of too strong breaking changes and a lot of small not very well explained xp-hp tweaks changes. Because of it I decided to base Afterlife Era on 1.16 in order to continue balancing without worrying too much about the mainline changes effects.
> 
> Balance of 1.14 may appeal more to Ladder era players because of different approaches to Elvish Shaman and Merman Hunter, for me 1.16 is more comfortable. I also like Dunefolk 1.16 more than Dunefolk 1.14, while ladder is generally indifferent to Dunefolk.
> 
> I agree that it makes sense to discuss changes of 1.18 in groups or individually and possibly leave some of them in place.
> 
I agree, but there are some strange stat distribution cases what are older than 1.18, 1.16 and even 1.14, that also need to be addressed.
> A brief summary of my opinions on the most important changes for me:
> 
> * A lot of changes to XP (and some HP) of level 1 and level 2 units. Those may seem pretty insignificant but they are pain for Ladder because many normal strategies are disrupted. I've seen complains about Loyalist Mage, also SkyEnd made a pretty good overview on the forums. So I'd prefer mostly going back to 1.16 here.
> * Saurian Soothsayer buffed MP and the subsequent removal from the Drakes leader pool, very strange buff.
>

> * I would also note increased MP of Elvish Outrider, feels like too much, he became faster than Gryphon Riders. That one also needs attention in regard of "Outride the Outriders" scenario of SoF.
>
That's actually the change I liked, Gryphon Rider has the advantage of flying, as does Drake Hurricane, I would rather increase MP of other mounted units, why would a Footpad run faster than horse, even Great Knight's, and decrease Outrider's damage, I don't see how can he shoot better than Champion.

> * Arcane rework. I don't really like it, it diminished arcane into "just another magical type of damage" and made a special note about it pretty much outdated. I would revert it or at least make it closer to 1.16: return 20% arcane for humans (and increased arcane res of the White Mage line and Paladin) and make it at least -30% for skeletons.
>
Agreed, but we should not forget other magical creatures who are vulnerable to arcane.

> * Orcish Crossbowman. It received a significant buff: very fast leveling to lvl 3 and increased melee. If not going back to 1.16 completely, I would nerf him back a bit, more xp to level, a bit less melee damage.
> 
> Probably forgot something, will add later if remember.
> 
> Obviously we should keep new Saurian Augurs 3 levels, and I believe keep the advancement of Orcish Slayer to Orcish Nightblade.

**Comment by Wedge009** [comment]
Wow, a lot has happened in just two days since I last checked.

> Also wouldn't reverting most/all of the 1.18 changes cause a huge amount of additional rework at this point?

> For what it's worth, I think it's crazy that people are talking about reverting to 1.16 stats.

I recognise I'm probably not 'qualified' to form a strong opinion on this subject as I focus solely on SP. I can see many others, on the other hand, have formed strong opinions justified or not. But _in principle_, I also think it's not a good idea to make a blanket revocation of all the unit development and balancing that has been done since 1.16 (or whatever preferred version one might have).

Not even being a MPer generally, but I observe (as an outsider) competitive games like DotA and Counter-Strike, perhaps even games like Smash Bros, that seem to have fairly regular updates to character/equipment balancing and such. Even changes that may be controversial at times seem to be accepted over time. I suppose one could argue such large productions have the expertise to draw upon as to whether a change is suitable or not.

**tl;dr** I think I would favour avoiding a blanket reversion in Wesnoth, but rather perhaps a focus on balancing the most controversial changes in 1.18 and making play-tested adjustments to those as needed.

Which brings me to another question: when/how would such adjustments be delivered? Would there be any changes to the current 1.18 release or would such changes only take effect in 1.19/1.20?

**Comment by ForestDragon-wesnoth** [comment]
> Which brings me to another question: when/how would such adjustments be delivered? Would there be any changes to the current 1.18 release or would such changes only take effect in 1.19/1.20?

Balance changes do not happen during stable versions (rightfully so otherwise it would break a lot of things), so any changes would be for 1.19/1.20

**Comment by ForestDragon-wesnoth** [comment]
> No, not really. Not for anything that truly _matters_. If it's something like "which of these optional new fauna units does the community want most", then sure. And if it's unrelated to the game design, then maybe it's fine to have a poll. Depends on the topic.

I agree, google form surveys should not become legitimized as a way to decide the game's fate on a large scale. even the forums at least have the advantage of people having to write an argument for why a change is good or bad rather than just clicking support/oppose.

**Comment by CelticMinstrel** [comment]
> why would a Footpad run faster than horse

I think it actually _is_ possible in theory for a human to outrun a horse. A trotting horse is only slightly faster than a running human, and while they can run quite a bit faster than a trot, they can't maintain that speed for long. So I don't think it's clear-cut whether a runner or rider would win in a marathon.

**Comment by gfgtdf** [comment]
> ... So I don't think it's clear-cut whether a runner or rider would win in a marathon.

There actually exists a ma vs horse marathon https://en.m.wikipedia.org/wiki/Man_versus_Horse_Marathon

**Comment by Wedge009** [comment]
It's fascinating what one can learn about real-life comparisons... for the purposes of Wesnoth game-play, however, I think the implication that a mounted unit can travel further/faster than a foot-unit (at least on even terrain) still stands. A sprinter in the real-life comparison isn't carrying armour/weapons, etc, and isn't expected to engage in combat at the end of the race. :p

**Comment by CelticMinstrel** [comment]
> A sprinter in the real-life comparison isn't carrying armour/weapons, etc, and isn't expected to engage in combat at the end of the race.

Not a sprinter, but other than that you make a good point. (A marathon is not a sprint. They're totally different things. Sprints are all about speed, whereas a marathon is more about endurance – going as fast as you can is not beneficial.)

Still, let's consider a couple of facts about the two specific units that were compared:

* The Grand Knight is an _armoured_ horse carrying a man in full plate armour.
* The Footpad is an _unarmoured_ man carrying nothing but a weapon and maybe a few supplies.

If you suppose that a man and a horse are roughly equal when unencumbered, then it seems obvious that the footpad should win in this comparison.

On the other hand, if you look at the Elvish Outrider, you see an _unarmoured_ horse carrying a man wearing _light armour_, equipped with _relatively_ light weapons (a bow and short sword). So you probably wouldn't expect a Footpad to beat an Outrider in speed, most of the time. (It might be possible, but not typical.)

**Comment by Wedge009** [comment]
> Not a sprinter...

I got the point, I think I got side-tracked by the discussion that a sprinter and a horse are comparable.

**Comment by Jonathan-Kelly** [comment]
After reading the worrying analysis of 1.18's balance provided above by @xSkyEndx it seems this discussion would do well to hear more from high ranked ladder players, since they are the most experienced individuals at multiplayer balance (which is generally the most sensitive to changes).

Also, their views may be underrepresented due to language barriers, as English language skills are almost a requirement for user made content creators and developers (most of us here are in these two categories) but mostly irrelevant to ladder competitors.

> I do feel that Ladder Era - and other modified default eras - represent a split in the community. Wesnoth has a limited amount of MP players, so in theory any division in the community makes games harder to find for both sub-communities. In addition, I assume that Ladder Era is very well-balanced. In theory, porting some of those changes over would also help make Default Era more balanced (though I'm sure the reality is a little more complex).

I concur.

**Comment by hladkyjakub** [comment]
Why is this marked as a good first issue?

**Comment by knyghtmare** [comment]
It's not

**Comment by hladkyjakub** [comment]
> It's not

When I go here, I see it there:
https://github.com/wesnoth/wesnoth/labels/Good%20first%20issue

**Comment by hladkyjakub** [comment]
Ouch, I haven't got the bottom part displayed.

**Comment by ForestDragon-wesnoth** [comment]
> I haven't ported any of my campaigns to 1.18 yet. Part of the reason I'm reluctant to is because of all the work that the balance changes will require. I support this proposal to the extent that it means less work for UMC maintainers.

Yeah, 1.18 was a massive change compared to 1.16 and way too much happened in one update. Previously changes happened much more slowly, I and I worry that 1.18 is becoming used as a precedent that huge changes to unit stats are the "new normal"

**Comment by ForestDragon-wesnoth** [comment]
For example with the arcane rework, while I can understand some people wanting skeleton vulnerability to be less than the old -50% arcane resistance, the fact that currently arcane is merely equal to impact/fire (-20%) feels wrong in my opinion, so a partial revert like -30% arcane for skeletons/zombies/liches could be a reasonable middle ground between 1.16 and 1.18

**Comment by Tezereth** [comment]
A bit late to the party, but I too would rather support a revert of stats to 1.16 or 1.14, even if only partial.
-After all, not every single change has been bad, albeit I disagree with most of the bigger ones, but it wouldn't be smart to not take a look at some of the smaller changes that didn't brought any problem and benefitted the unit or faction they were in without breaking the balance.

-On the "**reverting to 1.16 stats also mean adapting exisiting content again**" point I've seen, it really isn't a problem, because the "adaptation" already existed in 1.16, so it's adaptation based on a baseline that exist, which is noticeably easier than adaptation of entirely new stats with no previous baseline.

-For the arcane rework, I'd be one to advocate for removing the rework entirely, but I also understand some of the reasons that made this change happen. Reverting it to it's original stats for most units, like humans 20%, but not doing an entire revertion for undeads like skeleton and only going up to -30% is at least somewhat understandable to me. Though if the issue is "there's only limited options to use when fighting skeletons", I'll have to point out it is their whole point, to have only specific damage being useful against them. But again, if instead of being reverted to -50 or -40% it was -30%, I would understand.

**Comment by Jonathan-Kelly** [comment]
After much player feedback and some experiments, I think the gold costs of level 3 units need to reverted to their 1.16 or 1.14 values. The original values were a better reflections of their utility and the new values are too high (in some cases much too high).

We may be rediscovering just how well the original balance team knew what it was doing.

**Comment by Hejnewar** [comment]
> Ladder era
I honestly don't think that is possible. But that requires some deeper understanding and history behind it. Well let's say that for 1.14->1.16 balance team was composed out of that time pros (and me) that didn't like ladder era changes. Now there is another Balanced era, based on ladder era because again this time pro doesn't like the changes there. In short everyone including me knows best and there will always be discussions about what is correct.

> A bit late to the party 

Well I just got to know that this is even happening. Wish I known sooner, kinda feel left out but oh well.

**Comment by knyghtmare** [comment]
> After much player feedback and some experiments, I think the gold costs of level 3 units need to reverted to their 1.16 or 1.14 values. The original values were a better reflection of their utility and the new values are too high (in some cases much too high).

If the notion of stats reversion is on the common mind, I would suggest we put to the vote. 1.16, 1.14, (or, 1.18 without the godlike level 3 elves changes)? Actually, not yet. I got one topic.

One other thing I wish to convey here:

I think that the decisions of balancing units should not be undertaken by a singular entity. I have seen how we came to the present circumstances today.
Previously, we had 3 people. 
Now, I see just one. 
What happened? Where were the drafting of replacements taking place? Was it taking place though?

Thus, before we consider the notion of stats reversal, we should solve the problem that came before: the assignment of a balance team (preferable 5, minimum 3).

**Comment by Hejnewar** [comment]
Oh, I actually did try to fine new people so I asked around. Like IPS or matto, however in the end nothing came out of that (except for being asked for opinion about ageless from time to time by IPS I guess). 

Community had initial push to propose someone but I guess community didn't find anyone either as I haven't seen any proposal from anyone else.

**Comment by hladkyjakub** [comment]
I would for sure preffer to assing this task to a group of people. Not a pool voting about every change.
Voting is great thing but it's better to use it as a feedback than making decisions directly.

**Comment by Hejnewar** [comment]
Well we already have 2 people actually.

**Comment by CelticMinstrel** [comment]
> I would suggest we put to the vote

Reminder that voting is a bad method of decision-making. So no, we shouldn't put out the vote on "which version" to revert to.

(We shouldn't revert to _any_ version, though. Changing specific things back to how they were in a specific version is fine, if it's decided that specific change is bad. Reverting wholesale to a specific version is ridiculous.)

**Comment by Pentarctagon** [comment]
Also just to be clear about my own intentions with this issue: I don't intend to open any PR either partially or entirely rolling back any of the 1.18 balance changes. If people feel strongly that changes should be made, then they should make a PR implementing those changes.

In order that to have any chance at success rather than just being an exercise in frustration, I'd suggest:
* A list of different subsets of the 1.18 balance changes be made.
* Each specific item is discussed separately - I doubt there's any possibility of coming to a single state that everyone agrees on if all parts of it are discussed at the same time.

**Comment by ForestDragon-wesnoth** [comment]
> If people feel strongly that changes should be made, then they should make a PR implementing those changes.

alright, sounds reasonable

**Comment by cooljeanius** [comment]
Some PRs that are meant to address this: #9440 and #9438

**Comment by Gothyoba** [comment]
#9438 is not related to this by the way.

**Comment by cooljeanius** [comment]
> I will open a PR with this remedy.

Be sure to link it back to this issue if/when you do.

**Comment by Jonathan-Kelly** [comment]
Good point - it is now open as pull request #9468

**Comment by Pentarctagon** [comment]
Closing as there's a balancing team now and more specific issues/PRs being opened for balancing topics.
